packages/nc-gui/components/dashboard/settings/Erd.vue

```diff
@@ -0,0 +1,55 @@
+<script setup lang="ts">
```

Comment:
@bcakmakoglu The reason being that `ErdView` component is being reseted using key in these 2 Erd components, which solves all the state resetting related issues. Since vue flow store is only destroyed when that component is destroyed. 

So I made `ErdView` component such that it is only responsible for displaying data, while the config management/data reset is being done on the parent component.

Reply:
Mh... I understand why you wrap it in the `View` component but what I meant is there is a `Erd.vue` in `dashboard/settings` and almost the exact same `Erd.vue` component in `smartsheet-toolbar`

It just seems like duplicated code that could be merged into a single file that conditionally changes?
Or did I misunderstand you? :)

Reply:
@bcakmakoglu Got it 👍